tests/unit/objects/test_project_merge_request_approvals.py

```diff
@@ -168,6 +168,7 @@ def test_list_merge_request_approval_rules(project, resp_mr_approval_rules):
     assert len(approval_rules) == 1
     assert approval_rules[0].name == approval_rule_name
     assert approval_rules[0].id == approval_rule_id
+    repr(approval_rules)
```

Comment:
Is this a leftover from testing? :)

Reply:
It is not. Before this would raise an exception. Let me add a comment to it.

I did mention it in the commit message.

Reply:
Thanks, was confused by the lack of assert I think, and I missed that as I only checked the PR description :grin: